plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -3457,6 +3450,16 @@ else if (isMetadataColumnId(columnHandle.getId())) {
                 false));
     }
 
+    private static List<ManifestFile> getAllManifestsFromSnapshot(Table icebergTable, Snapshot snapshot)
```

Comment:
I woud rename to `readAllManifestsFromSnapshot` or `loadAllManifestsFromSnapshot`. 

https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#avoid-get-in-method-names-unless-an-object-must-be-a-java-bean